scripts/test-infra/verify.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash -e
+
+# Install deps
+curl -sfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh| sh -s -- -b $(go env GOPATH)/bin v1.27.0
```

Comment:
How about using version 1.30.0, á la https://github.com/kubernetes-sigs/node-feature-discovery/pull/349

Reply:
Hmm, you dropped golangci-lint. What was the reason for that?

Reply:
and this?

Reply:
> ...still 3 open conversations...



> ...still 3 open conversations...

Thanks for pointing me out. 
updated now